com.yubico.YubiOath.json

```diff
@@ -0,0 +1,257 @@
+		"/**"
+	    ],
+	    "sources": [
+		{
+                    "type": "archive",
+                    "url": "http://download.sourceforge.net/swig/swig-3.0.12.tar.gz",
+		    "sha256": "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+		}
+	    ]
+	},
+	{
+            "name": "pyscard",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/LudovicRousseau/pyscard/archive/release-1.9.7.tar.gz",
+		    "sha256": "402956d4c2c0ae265ff5d590efdf80297b07060fa923230658e682cff29a27a2"
+		},
+		{
+		    "type": "patch",
+		    "path": "pyscard-include-path.patch"
+		}
+	    ]
+	},
+	{
+            "name": "pyusb",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pyusb/pyusb/archive/v1.0.2.tar.gz",
+		    "sha256": "49dfa601f28e9399df62004170cfb623197ad32d8399885b7f789775767b50c3"
+		}
+	    ]
+	},
+	{
+            "name": "click",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pallets/click/archive/7.0.tar.gz",
+		    "sha256": "7cd752d4b5621bddf6428ca00ef5c2cd9b4b77bc41076a69eff51cbbfd4f8646"
+		}
+	    ]
+	},
+	{
+            "name": "asn1crypto",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/wbond/asn1crypto/archive/0.24.0.tar.gz",
+		    "sha256": "d2a82bb3c29e1abcb376b59679510b58dd796c813f8983f2db9b45e9ab228190"
+		}
+	    ]
+	},
+	{
+            "name": "cffi",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://files.pythonhosted.org/packages/64/7c/27367b38e6cc3e1f49f193deb761fe75cda9f95da37b67b422e62281fcac/cffi-1.12.2.tar.gz",
+		    "sha256": "e113878a446c6228669144ae8a56e268c91b7f1fafae927adc4879d9849e0ea7"
+		}
+	    ]
+	},
+	{
+            "name": "pycparser",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/eliben/pycparser/archive/release_v2.19.tar.gz",
+		    "sha256": "893ec3d0f26c0faa7d0b992979940353a1682f86b5cd1228740637055b112379"
+		}
+	    ]
+	},
+	{
+            "name": "cryptography",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"python3 setup.py install --prefix=/app"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pyca/cryptography/archive/2.6.1.tar.gz",
```

Comment:
latest is [2.7](https://github.com/pyca/cryptography/releases/tag/2.7)

Reply:
Fixed